solutions/landing-zone-v2/README.md

```diff
@@ -0,0 +1,585 @@
+
+ When deploying resources `kpt` will wait for all resources to deploy before finishing, `kpt` also provides annotations such as [depends-on](https://kpt.dev/reference/annotations/depends-on/) that allow us to order the deployment of resources. 
+
+ GitOps allows us to connect our cluster to a "source of truth" and deploy from that. To do this we use [Anthos Config Management](https://cloud.google.com/anthos/config-management), which is pre-installed in Config Controller, and this allows us to target either a [Git Repository](https://cloud.google.com/anthos-config-management/docs/concepts/configs) or [OCI Artifact](https://cloud.google.com/anthos-config-management/docs/how-to/publish-config-registry) (Container). 
+    
+ The benefit to this method is we now have an additional reconciliation process to ensure the deployed state matches our desired state or "source of truth". This also allows for improved automation and removes the need to run `kpt live apply` to deploy resources.
+
+ ## kpt
+    
+ Before deploying with `kpt` you will need to add `constraints.yaml` to the `.krmignore` file. This is due to needing to have the `constraintstemplate` resources deployed into the instance before the policy `constraint` can be deployed. Once the `constrainttemplates` have been deployed you can remove `constraints.yaml` from the `.krmignore` file and redeploy. This is not needed with either gitops deployment options.
+
+ Before running the `kpt` commands for the first time you will need to initialize the `kpt` package locally. You will only need to do this once after you initially get the `kpt` package
+
+ The following commands assume the `landing-zone` package is in your currently directory. If you are in the `landing-zone` directory you can remove omit the `landing-zone` argument from the commands.
+
+  ```bash
+    kpt live init landing-zone --namespace config-control
+  ```
+
+  ```bash
+    kpt fn render landing-zone
+    kpt live apply landing-zone --reconcile-timeout=2m --output=table
+  ```
+
+ The section around billing that looks like the following and can either be commented out or deleted.
+  ```bash
+    billingAccountRef:
+        # Replace "${BILLING_ACCOUNT_ID?}" with the numeric ID for your billing account
+        external: "${BILLING_ACCOUNT_ID?}" # kpt-set: ${billing-id}
+  ```
+
+ This will cause the project to spin up with no attached billing id and any service that requires billing to be enabled will pause deployment until billing is enabled. Billing can be added by a user with Billing User permission in the Billing UI. If you do not remove this section the project will fail to create.
+
+ You can view the status of any deployed object by running `kubectl get gcp` when connected to the Config Controller instance. If an object is pending or is displaying an error you can investigate by copying the name of the object and running the describe command.
```

Comment:
I am wondering if we need to specify the following flags if we have resources in multiple namespaces?

`kubectl get gcp -A` or `kubectl get gcp --all-namespaces` ?

Reply:
same as above, @fmichaelobrien @cartyc could you guys revise this section. I copied it from landing-zone v1 but never had the chance to test a deployment with kpt.